packages/protoc-gen-connect-react-query/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@connectrpc/protoc-gen-connect-query",
+  "name": "@connectrpc/protoc-gen-connect-react-query",
```

Comment:
Shouldn't we rename the npm binary as well?

Reply:
Ah right, yeah, good catch.